packages/compiler/lib/reflection.cadl

```diff
@@ -0,0 +1,12 @@
+namespace Cadl.Reflection;
+
+model ModelProperty {}
+model Interface {}
+model Enum {}
+model EnumMember {}
+model TemplateParameter {}
```

Comment:
What is the use case for this one?

Reply:
think I meant to remove that one but forgot